src/api/EscargotPublic.cpp

```diff
@@ -1146,6 +1146,45 @@ Evaluator::EvaluatorResult VMInstanceRef::executePendingJob()
 
 #ifdef ESCARGOT_DEBUGGER
 
+StringRef* DebuggerOperationsRef::BreakpointOperations::eval(StringRef* sourceCode, bool* isError)
+{
+    ExecutionState* state = toImpl(m_executionState);
+    Debugger* debugger = state->context()->debugger();
+
+    if (debugger == nullptr) {
+        if (isError != nullptr) {
+            *isError = true;
+        }
+
+        static const char errorMessage[] = "Debugger is not available";
+        return toRef(new Latin1String(errorMessage, sizeof(errorMessage) - 1));
+    }
+
+    if (isError != nullptr) {
+        *isError = false;
+    }
+
+    String* result;
+
+    debugger->setStopState(ESCARGOT_DEBUGGER_IN_EVAL_MODE);
+
+    try {
+        Value asValue(toImpl(sourceCode));
+        Value evalResult(Value::ForceUninitialized);
+        evalResult = state->context()->globalObject()->evalLocal(*state, asValue, state->thisValue(), reinterpret_cast<ByteCodeBlock*>(weakCodeRef())->m_codeBlock, true);
+        result = evalResult.toStringWithoutException(*state);
```

Comment:
The return value of this `eval` function should be String value?

Reply:
Debugger usually interacts with user, and uses strings for that. But we can change that of course. Which one you prefer?
1) Keep the current one
2) Use value result
3) Pass a boolean which optionally converts the result to string

Reply:
I see. current version looks good :)